### PR TITLE
[LP-1667] Added new command to fetch course structures

### PR DESCRIPTION
**Pull request by saad-waqar**
Added new command to fetch course structures. it takes course ids as input and returns course_structures for only those ids which are valid

### CODE REVIEW

File: common/djangoapps/philu_commands/management/commands/fetch_course_structures.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+from opaque_keys.edx.keys import CourseKey
+from opaque_keys import InvalidKeyError
+from philu_commands.helpers import generate_course_structure
+
+log = getLogger(__name__)
+
+
+class Command(BaseCommand):
+    help = """
+    This command returns the course structure for all the course ids given in arguments
```
Comment (zeshanarifios) [inline common/djangoapps/philu_commands/management/commands/fetch_course_structures.py]: I think 'This command prints' or `This command displays' would be more appropriate.

File: common/djangoapps/philu_commands/management/commands/fetch_course_structures.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    example:
+        manage.py ... fetch_course_structures course_id_1 course_id_2
+    """
+
+    def add_arguments(self, parser):
+        parser.add_argument('course_ids', nargs='+', help='Course ids for which we require the course structures.')
```
Comment (zeshanarifios) [inline common/djangoapps/philu_commands/management/commands/fetch_course_structures.py]: This line is too long.

File: common/djangoapps/philu_commands/management/commands/fetch_course_structures.py [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+    def handle(self, *args, **options):
+        course_structures = []
+        course_ids = options['course_ids']
+
+        for course_id in course_ids:
+            try:
+                course_key = CourseKey.from_string(course_id)
+            except InvalidKeyError:
+                log.error("Invalid course id provided: {}".format(course_id))
```
Comment (zeshanarifios) [inline common/djangoapps/philu_commands/management/commands/fetch_course_structures.py]: please make quotes consistent , As we are using single quotes in most of places, So here double quotes can be replaced with single quotes.

File: common/djangoapps/philu_commands/management/commands/fetch_course_structures.py [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+                continue
+
+            course_structure = generate_course_structure(course_key)
+            course_structure['course_id'] = course_id
```
Comment (zeshanarifios) [inline common/djangoapps/philu_commands/management/commands/fetch_course_structures.py]: What about moving this line inside ```generate_course_structure```

File: common/djangoapps/philu_commands/management/commands/fetch_course_structures.py (hunk 2 above)
Comment (saad-waqar) [reply common/djangoapps/philu_commands/management/commands/fetch_course_structures.py]: We're not using any line length limitation in PhilU yet

File: common/djangoapps/philu_commands/management/commands/fetch_course_structures.py (hunk 4 above)
Comment (saad-waqar) [reply common/djangoapps/philu_commands/management/commands/fetch_course_structures.py]: what purpose would that serve us?

**Pull request merged**